src/dodal/devices/electron_analyser/specs/driver_io.py

```diff
@@ -17,12 +17,16 @@
 )
 from dodal.devices.electron_analyser.abstract.types import TLensMode, TPsuMode
 from dodal.devices.electron_analyser.specs.enums import AcquisitionMode
-from dodal.devices.electron_analyser.specs.region import SpecsRegion
+from dodal.devices.electron_analyser.specs.region import PassEnergy, SpecsRegion
 
 
 class SpecsAnalyserDriverIO(
     AbstractAnalyserDriverIO[
-        SpecsRegion[TLensMode, TPsuMode], AcquisitionMode, TLensMode, TPsuMode
+        SpecsRegion[TLensMode, TPsuMode],
+        AcquisitionMode,
+        TLensMode,
+        TPsuMode,
+        PassEnergy,
```

Comment:
nit: this would be less confusing to me if we just used `float`

```suggestion
        float,
```